app/views/users/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here is a list of all users</h1>
+
+<h1>List of all the users</h1>
+
+<div class="wrapper">
+  <h1>Users</h1>
+  <ul>
+    <% @users.each do |user| %>
+      <li class="userNameWrapper">
+      <div><img alt="Profile P."/></div>
+      <div class="userName">
+      <%= link_to user_path(user), class: 'userNameAnchor' do %>
+        <h2><%= user.name %></h2>
+        <h6>Number of posts: <%= user.posts.count %></h6>
+          <% end %>
+        </div>
```

Comment:
- [ ] Great job working on the functionality so far but I think it would be great if you fix the layout a little bit to match the given layout since the username has more width in the given design so kindly fix it 🔥🚀🤩


- Your design
![image](https://github.com/Jrmcuervo/Blog-app_ror/assets/58553711/84735d05-53d3-47d4-9d0c-54df21b50366)


- Given
![image](https://github.com/Jrmcuervo/Blog-app_ror/assets/58553711/bf646aa6-a16c-4b6a-8627-cecbc6c0b269)